examples/ec/configuration_hcl_ecdeployment.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: terraform.core.oam.dev/v1beta1
+kind: Configuration
+metadata:
+  name: ec-deployment
+spec:
+  hcl: |
+    data "ec_stack" "latest" {
+      version_regex = "latest"
+      region        = var.ec_region
+    }
+
+    resource "ec_deployment" "project" {
+      name = var.project_name
+
+      region                 = var.region
+      version                 = data.ec_stack.latest.version
+      deployment_template_id = "gcp-io-optimized"
+
+      elasticsearch {
+        autosccale = "true"
```

Comment:
➜  hcl git:(add_ec_support) ✗ t apply
╷
│ Error: Unsupported argument
│
│   on deployment.tf line 23, in resource "ec_deployment" "project":
│   23:     autosccale = "true"
│
│ An argument named "autosccale" is not expected here. Did you mean "autoscale"?

Reply:
Oops! I just pushed a fix for this typo.

Reply:
Yes, made a mistake here. I have pushed a fix.